### PR TITLE
Added a summer school for this year

### PR DESCRIPTION
**Pull request by duarte-jfs**

### CODE REVIEW

File: FEEDBACK.md [hunk 1]
```diff
@@ -24,14 +24,37 @@ Additional Reviews:
 
 
 ## Programs
+### Los Alamos Quantum Computing Summer School (ISTI)
```
Comment (1ethanhansen) [inline FEEDBACK.md]: This already exists in the repo, so this is creating a merge conflict. It looks like your fork is behind my master branch a bit. Either catch up your branch, then re-add the summer school, or you can just re-fork my repo and re-add the summer school

File: FEEDBACK.md [hunk 2]
```diff
@@ -24,14 +24,37 @@ Additional Reviews:
...
+Additional Reviews:
+
 ### QCSYS (University of Waterloo IQC)
 |Links|  |   |   |
 | :--: | :--: | :--: | :--: |
 
 Additional Reviews:
 
+
+### QOSF Mentorship Program (Quantum Open Source Foundation)
```
Comment (1ethanhansen) [inline FEEDBACK.md]: This already exists in the repo, so this is creating a merge conflict. See above comment

File: FEEDBACK.md [hunk 3]
```diff
@@ -24,14 +24,37 @@ Additional Reviews:
...
+|Links|  |   |   |
+| :--: | :--: | :--: | :--: |
+
+Additional Reviews:
+
+### Summer School on Machine Learning and Big Data with Quantum Computing (University of Porto)
```
Comment (1ethanhansen) [inline FEEDBACK.md]: This lgtm :)

File: README.md [hunk 4]
```diff
@@ -21,9 +21,14 @@ A list of hands-on opportunites (not jobs) in quantum computing
 ## Programs
 | Name | Link | Organization | Date | Status | Feedback |
 | :--: | :--: | :--: | :--: | :--: | :--: |
+| Los Alamos Quantum Computing Summer School| [Info Page](https://www.lanl.gov/projects/national-security-education-center/information-science-technology/summer-schools/quantumcomputing/index.php) | Information Science and Technology Institute (ISTI) | Summer (10 weeks) | ![closed status][closed] | [![FB][feedback]](FEEDBACK.md#los-alamos-quantum-computing-summer-school) |
```
Comment (1ethanhansen) [inline README.md]: This already exists in the repo, so this is creating a merge conflict. See longer comment in FEEDBACK.md.

File: README.md [hunk 5]
```diff
@@ -21,9 +21,14 @@ A list of hands-on opportunites (not jobs) in quantum computing
...
 | QCSYS | [Info Page](https://uwaterloo.ca/institute-for-quantum-computing/programs/qcsys) | University of Waterloo IQC | August | ![closed status][closed] | [![FB][feedback]](FEEDBACK.md#qcsys-university-of-waterloo-iqc) |
+| QOSF Mentorship program | [Info Page](https://qosf.org/qc_mentorship/) | Quantum Open Source Foundation | Program runs in batches | ![closed status][closed] | [![FB][feedback]](FEEDBACK.md#qosf-mentorship-program) |
```
Comment (1ethanhansen) [inline README.md]: This already exists in the repo, so this is creating a merge conflict. See longer comment in FEEDBACK.md.

File: README.md [hunk 6]
```diff
@@ -21,9 +21,14 @@ A list of hands-on opportunites (not jobs) in quantum computing
...
+| Summer School on Machine Learning and Big Data with Quantum Computing | [Info Page](https://smbq2020.dcc.fc.up.pt/) | University of Porto | September 7-8 2020 | ![open status][open] | [![FB][feedback]](FEEDBACK.md#SMBQ2020) |
```
Comment (1ethanhansen) [inline README.md]: The link to FEEDBACK is wrong. Try going to FEEDBACK.md and clicking on the link button next to the header, then copypasting the last bit of the url that pops up in your browser

**Review by 1ethanhansen — CHANGES_REQUESTED** [review]

**Comment by duarte-jfs** [comment]
I really have to look into how to work with github, I'll take a look at it tomorrow. I need to fix this, should have seen I was 5 commits behind

**Comment by 1ethanhansen** [comment]
> 
> 
> I really have to look into how to work with github, I'll take a look at it tomorrow. I need to fix this, should have seen I was 5 commits behind

It's all good :+1: It's super cool you're learning. Feel free to shoot me an email at [1ethanhansen@protonmail.com](mailto:1ethanhansen@protonmail.com) if you want to walk through it with me, I'm always happy to help :)

**Comment by 1ethanhansen** [comment]
Hi @duarte-jfs any updates on this? 😄

**Comment by duarte-jfs** [comment]
@1ethanhansen  Hi, yeah, you should have already an up to date pull request. Sorry for the delay, it completely slipped by me.

**Pull request closed without merging**